doc/source/index.rst

```diff
@@ -274,6 +274,7 @@ Papers
    raysgd/raysgd_pytorch.rst
    raysgd/raysgd_tensorflow.rst
    raysgd/raysgd_dataset.rst
+   raysgd/raysgd_ptl.rst
```

Comment:
fine for now, but I think we will want to refactor the raysgd documentation soon